www/static/css-src/main.scss

```diff
@@ -194,6 +194,23 @@ code {
     color: #000000;
 }
 
+/* algolia search formatting */
+.algolia-docsearch-suggestion--category-header, .algolia-docsearch-suggestion {
+    font-family: "Raleway", Helvetica, Arial, sans-serif !important;
+}
+
+.algolia-docsearch-suggestion--category-header {
+    background-color: #262626   !important;
+    font-size: 16px             !important;
+    font-weight: bold           !important;
+    line-height: 30px           !important;
+}
+
+.algolia-docsearch-suggestion {
+    font-size: 14px     !important;
+    font-weight: 300    !important;
+}
+
```

Comment:
I just tried this guy in my browser, and it looks like there are just a few inconsistencies with the current styling.

Old:
![desktop_old](https://cloud.githubusercontent.com/assets/405489/14659626/833ffdcc-0652-11e6-91a8-c3d3c485f58a.PNG)
![mobile_old](https://cloud.githubusercontent.com/assets/405489/14659627/8821ed64-0652-11e6-96a5-b16835f124ab.PNG)

New:
![desktop_new](https://cloud.githubusercontent.com/assets/405489/14659632/8f7d2880-0652-11e6-9bfb-bcf9575666b8.PNG)
![mobile_new](https://cloud.githubusercontent.com/assets/405489/14659634/93f0b198-0652-11e6-9b8c-3a3282365672.PNG)